imports/ui/components/board/board.jsx

```diff
@@ -6,8 +6,9 @@ import '/imports/ui/styles/_board.scss';
 import { categories } from '/lib/data/categories.js';
 
 class Board extends React.Component {
-  constructor(prop) {
-    super(prop);
+  constructor(props) {
+    super(props);
+    console.log(props) //comes from BoardContainer
```

Comment:
Let's try our best not to commit `console.log`s in ;) -- makes it harder to clean up when being pulled elsewhere